### PR TITLE
Add Dropout layers to VGG models

### PR DESCRIPTION
**Pull request by unixnme**
These models should have Dropout layers, but somehow missing from Keras.
reference: http://www.robots.ox.ac.uk/~vgg/research/very_deep/

### CODE REVIEW

**Comment by fchollet** [comment]
We don't include regularizers in the pre-trained applications.
